controller/setting_controller.go

```diff
@@ -1853,6 +1857,10 @@ func (info *ClusterInfo) collectNodeScope() {
 		info.logger.WithError(err).Warn("Failed to collect host OS distro")
 	}
 
+	if err := info.collectLonghornImageRegistry(); err != nil {
```

Comment:
Should we collect this info per node or for per cluster?

Reply:
cluster level.

Reply:
Updated. Thank you.